charge_backend/callback_logger.py

```diff
@@ -17,13 +18,14 @@ async def handle_callback_log(message):
         # timestamp = record["time"].isoformat(" ", timespec='seconds')
         msg = record["message"]
         level = record["level"].name
+        if not source:
+            source = f"Logger ({level})"
```

Comment:
```suggestion
            LEVELS = {
               'DEBUG': 'Debug',
               'VERBOSE': 'Verbose',
               'INFO': 'Info',
               'WARN': 'Warning',
               'WARNING': 'Warning',
               'ERROR': 'Error'
            }
            level_str = LEVELS.get(level, level)
            source = f"Logger ({level_str})"
```
how about something like this?